app/models/case/sar/offender.rb

```diff
@@ -131,6 +132,32 @@ def validate_third_party_names
     errors[:third_party_name].any? || errors[:third_party_company_name].any?
   end
 
+  def validate_recipient
+    if !third_party && recipient == 'third_party_recipient'
```

Comment:
I don't think we should have the `!third_party` as part of the conditional here - there are scenarios where the requester can be the data subject but we still have a third_party_recipient

Did you find we had to have it because of something failing?